server/src/test/java/io/crate/integrationtests/TransportSQLActionTest.java

```diff
@@ -1975,7 +1975,7 @@ public void test_bit_string_can_be_inserted_and_queried() throws Exception {
         // primary key lookup uses different execution path to decode the value
         execute("select xs from tbl where id = 6");
         assertThat(TestingHelpers.printedTable(response.rows()), is(
-            "B'1100'\n"
+            "B'1001'\n"
```

Comment:
insert looks like    
`execute("insert into tbl (id, xs) values (1, B'0000'), (2, B'0001'), (3, B'0011'), (4, B'0111'), (5, B'1111'), (6, B'1001')");`
fixing a typo in assertion